### PR TITLE
typos

### PR DESCRIPTION
**Pull request by retokromer**

### CODE REVIEW

**Comment by mbunkus** [comment]
Thanks.

File: specification.markdown [hunk 1]
```diff
@@ -10,7 +10,7 @@ EBML uses a simple approach of building Elements upon three pieces of data (tag,
 
 ## Notation and Conventions
 
-The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [@!RFC2119](https://tools.ietf.org/html/rfc2119).
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC2119](https://tools.ietf.org/html/rfc2119).
```
Comment (dericed) [inline specification.markdown]: Just a note that the `@!` is used by [mmark](https://github.com/miekg/mmark/wiki/Syntax#citations) in the process to create the RFC output. We'll somehow need to find a compromise to support the auto-creation of the references section that works for each output.

Comment (mbunkus) [inline specification.markdown]: I wasn't aware of that. Sorry for merging so quickly without waiting for feedback first.

One possibility is to leave those markers in and to include a small script in the build process that removes them (they're very distinct and easy to grab via regexes) when outputting to a format that doesn't use them.

Comment (retokromer) [inline specification.markdown]: Hmm... there SHOULD be a solution ;-) I'll try to find out.

Comment (retokromer) [inline specification.markdown]: It seems that by declaring the `@!` as markdown comment it doesn't work. The best is possibly to revert and keep only the Class D typo/error.

**Comment by mbunkus** [comment]
Reverted.

**Comment by retokromer** [comment]
Thank you! And sorry for the confusion...

**Pull request merged**